### PR TITLE
Mayhem

### PR DESCRIPTION
**Pull request by ennamarie19**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#Build Stage
```
Comment (xansec) [inline Dockerfile]: Please do not use git clone or `ldd`.

File: Mayhemfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+image: ghcr.io/ennamarie19/openslide-fuzz:latest
```
Comment (xansec) [inline Mayhemfile]: Please omit image name.

File: test/fuzz.c [hunk 3]
```diff
@@ -0,0 +1,76 @@
+#include <stdint.h>
```
Comment (xansec) [inline test/fuzz.c]: can we rename this to indicate what's being fuzzed?

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Comment by capuanob** [comment]
@xansec Fixed in a new PR, this wasn't appearing for some reason. Sorry!

**Pull request merged**